.eslintrc.js

```diff
@@ -11,6 +11,8 @@ module.exports = {
   rules: {
     'consistent-return': 'off',
     'jsdoc/require-jsdoc': 'off',
+    'jsdoc/tag-lines': 'off',
```

Comment:
This rule was tell me that

```js
/**
 * A description.
 *
 * @param {string} foo
 */
```

is invalid because it wants the block to look like:

```js
/**
 * A description.
 * @param {string} foo
 */
```

Personally, I find that very ugly. I much prefer having the separation between the introductory text and the parameter (and other tags) defintions.